runtimes/pythonrt/py_test.go

```diff
@@ -78,3 +78,29 @@ func processEnv(t *testing.T, pid int) map[string]string {
 	}
 	return env
 }
+
+var pyVersionCases = []struct {
+	version string
+	major   int
+	minor   int
+	err     bool
+}{
+	{"Python 3.12.2", 3, 12, false},
+	{"Python 2.7.18", 2, 7, false},
+	{"Python 3", 0, 0, true},
+	{"", 0, 0, true},
```

Comment:
3 more tests?
- "Python" , e.g. starts with right string, but there is no version at all
- "Python X.X.X more stuff"
- "meow X.X.X"